src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,105 @@
+package duke;
+
+
+import exceptions.DukeException;
+import tasks.*;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Parser {
+    static final int TODOLENGTH = 4;
+    static final int EVENTLENGTH = 5;
+    static final int DEADLINELENGTH = 8;
+    static final int DELETELENGTH = 6;
```

Comment:
Consider adding an underscore to differentiate between one word and the next. For example, 
`private static final int TODO_LENGTH = 4`